nodeEditor/src/diagram/frameNodePort.ts

```diff
@@ -72,7 +72,11 @@ export class FrameNodePort extends NodePort {
         if (!displayManager || displayManager.shouldDisplayPortLabels(block)) {
             let portLabel = root.ownerDocument!.createElement("div");
             portLabel.classList.add("port-label");
-            portLabel.innerHTML = connectionPoint.displayName || connectionPoint.name;        
+            let portName = connectionPoint.displayName || connectionPoint.name;
+            if (portName === "output") {
```

Comment:
What if I want to name it "output" ?

Reply:
I did not think of that use case. That was silly of me. Updated.